### DISCUSSION TITLE
Add "AdGuard Tracking Protection" back to the lists

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2456] [General]
This list was removed just because of a placeholder being displayed on a single site on the entire Internet: https://github.com/gorhill/uBlock/commit/eac8e7ed62983ae970d9ab346a5b394b5c412053

However, the underlining issue has already been fixed and ameshkov apologized for it: https://github.com/AdguardTeam/AdguardFilters/issues/139081#issuecomment-1380603033.

**Comment by gothic-bum** [comment]
> This list was removed just because of a placeholder being displayed on a single site on the entire Internet:

See this comment:  
https://github.com/AdguardTeam/AdguardFilters/issues/139081#issuecomment-1382878362

**Comment by Lurux** [comment]
@gorhill, is there any particular reason you don't want to add this list back ? Has it often been a problem in the past ?

This issue was solved months ago, and it seems the Adguard folks have already apologized for the miscommunication / mishandling of the issue and are [set to not repeat this error again](https://github.com/AdguardTeam/AdguardFilters/issues/139081#issuecomment-1380603033).

I like to use this list as a second layer of protection on top of the Easylist one, but now I have to manually add the list on every new install I do - and I always struggle to find the link back.

Is there any hope to have the list back ?

**Comment by uBlock-user** [comment]
> Is there any hope to have the list back ?

gorhill will decide eventually..

**Reply by Yuki2718** [reply]
That's a bit overstatement, you know.

**Reply by uBlock-user** [reply]
@Yuki2718 I must have forgotten then, did he say he's bringing the mentioned list back ?

**Reply by Yuki2718** [reply]
https://github.com/orgs/uBlockOrigin/teams/ublock-filters-volunteers/discussions/459/comments/39

**Reply by uBlock-user** [reply]
Edited.

**Reply by Lurux** [reply]
> https://github.com/orgs/uBlockOrigin/teams/ublock-filters-volunteers/discussions/459/comments/39

Link dead ? (404 for me)

**Reply by iam-py-test** [reply]
It is a private discussion only accessible to people part of @ublockorigin (i.e. ublock-user), but people like you and me don't have access and thus get a 404

**Reply by Lurux** [reply]
Could someone please tell us what it says, then ?

**Reply by Yuki2718** [reply]
> Could someone then tell us what it says, please ?

No. private discussion is to discuss privately.

**Reply by Lurux** [reply]
@Yuki2718, I didn't ask for the whole readout, just the reason why the list will / will not be re-added. If there's a particular reason behind it, I think people here would appreciate to know.

That being said, I originally didn't see the comment was edited, hence my confusion reading this. Now reading the conversation with that in mind, I understand the decision just hasn't been made yet, it that correct ?

<details><summary>Side note</summary>
<br/>
Just so you don't think I'm completely dumb - Yes, I did see the "Edited." comment by @uBlock-user, but because the "edited" button isn't in the same place as in issue comments, I was left wondering what edited thing he was referring to. Please don't judge me, I'm just completely dumb.
</details>

**Reply by krystian3w** [reply]
@gwarser Are you planning migrate private team chat to private repository?

![image](https://user-images.githubusercontent.com/35370833/227181227-b9aec2da-a723-4bf0-a48b-14624918321e.png)

example: https://user-images.githubusercontent.com/35370833/227181579-ae50e33b-3d2a-41bc-94ff-e26b6f875c1f.png, https://user-images.githubusercontent.com/35370833/227182084-6a1f46b6-e1c8-4eeb-99ee-ee893803ea8b.png

The raw format version may look like IRC chat (after May 8, 2023) if we decide not to migrate (photos will probably turn into raw links).

**Reply by Yuki2718** [reply]
> Are you planning migrate private team chat to private repository?

Already done

**Comment by uBlock-user** [comment]
https://github.com/gorhill/uBlock/commit/02dff78eab1c62e49829b3970626bf43b919f496
